src/services/review/ReviewRequestService.ts

```diff
@@ -1,34 +1,60 @@
 import { AxiosResponse } from "axios"
 import _ from "lodash"
-import { errAsync, okAsync, ResultAsync } from "neverthrow"
+import {
+  err,
+  errAsync,
+  ok,
+  okAsync,
+  Result,
+  ResultAsync,
+  combine,
+} from "neverthrow"
```

Comment:
have we decided to use neverthrow across our codebase?

Reply:
I think is an interesting way to handle errors, but it's quite a shift in the way conventional errors are handled in js

Reply:
agreed, but throwing isn't good because then devs aren't inclined to handle errors properly + there's no oversight onto what errors are actually being returned. 

this forces errors to be explicit and handled properly rather than throwing and expecting a middleware to do so somewhere (as is done now)

Reply:
back to the discussion the team had on using neverthrow, I guess we will be postponing the use of the library till we get clarity on some of the refactorings we need to do in our code right?